From c286e563b587bab7cbbfa650c64af95d8e2db1d6 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 5 Dec 2024 12:08:23 +0000
Subject: [PATCH] serial: rpi-fw-uart: Demote debug log messages

A dev_info call in rpi_fw_uart_configure causes kernel log output every
time one opens the UART. Demote it to dev_dbg.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/tty/serial/rpi-fw-uart.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/tty/serial/rpi-fw-uart.c
+++ b/drivers/tty/serial/rpi-fw-uart.c
@@ -278,9 +278,9 @@ static int rpi_fw_uart_configure(struct
 	dev_dbg(port->dev, "version %08x, reg addr %x\n", msg.version,
 		msg.fifo_reg_base);
 
-	dev_info(port->dev, "started %d baud %u data %u stop %u rx %u tx %u flags %u fifosize %u\n",
-			msg.start, msg.baud, msg.data_bits, msg.stop_bits,
-			msg.gpio_rx, msg.gpio_tx, msg.flags, msg.fifosize);
+	dev_dbg(port->dev, "started %d baud %u data %u stop %u rx %u tx %u flags %u fifosize %u\n",
+		msg.start, msg.baud, msg.data_bits, msg.stop_bits,
+		msg.gpio_rx, msg.gpio_tx, msg.flags, msg.fifosize);
 
 	if (msg.fifosize != port->fifosize) {
 		dev_err(port->dev, "Expected fifo size %u actual %u",
